test/e2e/address-book.spec.js

```diff
@@ -55,61 +55,51 @@ describe('MetaMask', function () {
   describe('Going through the first time flow', () => {
     it('clicks the continue button on the welcome screen', async () => {
       await driver.findElement(By.css('.welcome-page__header'))
-      const welcomeScreenBtn = await driver.findElement(By.xpath(`//button[contains(text(), '${enLocaleMessages.getStarted.message}')]`))
```

Comment:
.... huh. That was a surprise. The e2e test failure on this line was _precisely_ what these changes were meant to prevent 🤦‍♂ 

Maybe the element _was_ visible momentarily, then was obscured _after_ the driver had passed that condition? 🤔

Reply:
Well, at least it only failed once 😬